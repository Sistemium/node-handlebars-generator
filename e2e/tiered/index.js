// run basic end-to-end test

var HandlebarsGenerator = require('../../');

HandlebarsGenerator.registerSourceDirectory(__dirname + '/src');

HandlebarsGenerator.registerSourceDirectory(__dirname + '/lib', {
	prefix: 'comps'
});

HandlebarsGenerator.registerPage('index', 'main', {});

HandlebarsGenerator.registerPage('help/faq', 'help/faq', {});

HandlebarsGenerator.generatePages(__dirname + '/dist')
	.then(function (r) {
		console.log('generated pages');
	}, function (e) {
		console.error('failed to generate pages:', e);
	})
;
