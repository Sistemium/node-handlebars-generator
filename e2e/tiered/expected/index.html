Home Landing

button
copyright (c) 2016